devito/operations/interpolators.py

```diff
@@ -256,7 +256,8 @@ def callback():
 
             # Write/Incr `self`
             lhs = self.sfunction.subs(self_subs)
-            last = [Inc(lhs, rhs)] if increment else [Eq(lhs, rhs)]
+            last = [Inc(lhs, rhs, implicit_dims=implicit_dims)] if increment else \
```

Comment:
imho more readable as:

```
if increment:
    cls = Inc
else:
    cls = eq
last = [cls(lhs, rhs, ...)]
```

Reply:
Yeah thought about doing it and forgot before pushing will fix